lib/elixir/pages/unicode-syntax.md

```diff
@@ -34,19 +34,21 @@ Elixir does not allow the use of ZWJ or ZWNJ in identifiers and therefore does n
 
 ### Atoms
 
-Unicode atoms in Elixir follow the identifier rule above with the following modifications:
+Unquoted unicode identifier-based atoms in Elixir follow the identifier rule above with the following modifications:
```

Comment:
I think this is more precise but without specifying that it is, I don't think it helps the user understand it. So I would propose to revert it and clarify it below.

Reply:
```suggestion
Unicode atoms in Elixir follow the identifier rule above with the following modifications:
```